radar-auth/src/main/java/org/radarcns/auth/config/YamlServerConfig.java

```diff
@@ -38,15 +37,7 @@
      * @return The initialized configuration object based on the contents of the configuration file
      * @throws ConfigurationException If there is any problem loading the configuration
      */
-    public YamlServerConfig loadConfig() {
-        if (config == null) {
-            config = readFromFileOrClasspath();
-        }
-        return config;
-    }
-
-
-    private YamlServerConfig readFromFileOrClasspath() {
+    public static synchronized YamlServerConfig readFromFileOrClasspath() {
```

Comment:
because there is no shared data between calls (no variables are modified), the `synchronized` keyword is not necessary here. It's only necessary if a reference to the `config` variable is kept.

Reply:
Yes. Understood. sorry. I thought I removed the synchronized here. I added it when I had the variable and forgot to remove.